docs/architecture/cloud-native/azure-data-storage.md

```diff
@@ -181,7 +181,7 @@ To partition the container, items are divided into distinct subsets called [log
 
 **Figure 5-18**: Cosmos DB partitioning mechanics
 
-Note in Figure 5-18 how each item includes a partition key of either ‘city’ or ‘airport’. This partition key determines the item’s logical partition. Each city code is assigned to a logical partition in the container on the left-side and those with an airport code to the container on the right. Combining the partition key value with an item's ID value creates the item's index, which uniquely identifies the item.
+Note in Figure 5-18 how each item includes a partition key of either `city` or `airport`. This partition key determines the item's logical partition. Each city code is assigned to a logical partition in the container on the left-side and those with an airport code to the container on the right. Combining the partition key value with an item's ID value creates the item's index, which uniquely identifies the item.
```

Comment:
I feel like the backticks is more suitable here. Can someone confirm ?
I've done the same in other few places.

Reply:
As part of this PR https://github.com/dotnet/docs/pull/15353, this content is moved to a new page `relational-vs-nosql-data`. @robvet I think, backticks is correct. Can you consider this change in the new PR?